src/components/pages/TeachersLessonOverview/OverviewPresentation.tsx

```diff
@@ -20,12 +21,12 @@ const OverviewPresentation: FC<OverviewPresentationProps> = ({
   const slidesId = asset.split("/")?.[5];
   const isWorksheetPortrait = !isWorksheetLandscape && isWorksheet;
   return (
-    <Box $ba={[3]} $width={"100%"}>
+    <Box $width={"100%"}>
       <AspectRatio ratio={isWorksheetPortrait ? "2:3" : "16:9"}>
         <iframe
           src={`https://docs.google.com/presentation/d/${slidesId}/embed?start=false&amp;loop=false&amp;delayms=3000`}
           title={`slide deck: ${title}`}
-          width="100%"
+          width="101%"
```

Comment:
Actually this reverts a change I made after discussion with Monica and Jack. The border shoulkd be solid and not use box borders. Please can you put it back

Reply:
haha
ok no problem